docs/embeddable-uis/troubleshooting.mdx

```diff
@@ -0,0 +1,125 @@
+---
+sidebar_position: 6
+title: Troubleshooting
+---
+
+In this page, we will cover some common issues that you might encounter while embedding Permit Elements in your application.
+
+:::tip
+In most cases, we recommend using **"Developer Tools" -> "Network"** (tab) in your browser to debug the requests and responses of the login method or `/me` requests.
+![Network tab](/img/elements/troubleshooting/devtools.png)
+:::
+
+## General Issues
+Before diving into the specific issues, here are some general issues that you might encounter while embedding Permit Elements in your application.
+- Make sure the Element's URL you set in the `<iframe>` tag is correct - Use the "Generate Code" button from the Element's configuration page.
```

Comment:
Check if you can add link here

Reply:
It's quite hard to add a link here to your specific element :/